core/debugger/remote_debugger.cpp

```diff
@@ -529,6 +530,26 @@ void RemoteDebugger::debug(bool p_can_continue, bool p_is_error_breakpoint) {
 			} else if (command == "set_skip_breakpoints") {
 				ERR_FAIL_COND(data.is_empty());
 				script_debugger->set_skip_breakpoints(data[0]);
+			} else if (command == "evaluate") {
+				String expression_str = data[0];
+				int frame = data[1];
```

Comment:
```suggestion
				ERR_CONTINUE(data.size() != 2);
				ERR_CONTINUE(data[0].get_type() != Variant::STRING);
				ERR_CONTINUE(data[1].get_type() != Variant::INT);
				String expression_str = data[0];
				int frame = data[1];
```

Avoid crashing when invalid data is received